.github/workflows/build_ubuntu.yml

```diff
@@ -146,12 +146,6 @@ jobs:
         run: |
           cmake --build _build --config Release -j$(nproc)
 
-      - name: Test
-        run: |
-          export PATH=${GITHUB_WORKSPACE}/_build/vcpkg_installed/x64-linux/tools/openmpi/bin:$PATH
-          cd _build
-          ctest -C Release --output-on-failure -L "medium|unit|benders|lpnamer"
-
```

Comment:
In my opinion, you should run unit tests as well a "full run" tests. Just because "full run" test don't return errors doesn't mean there is no regression.

Reply:
can't run tests with the executable,

Reply:
Yes this is an unwanted modification